app/assets/javascripts/govuk_publishing_components/analytics-ga4/init-ga4.js

```diff
@@ -0,0 +1,27 @@
+var initFunction = function () {
```

Comment:
Should this have it's own:

 ```
 window.GOVUK = window.GOVUK || {}
window.GOVUK.analyticsGA4 = window.GOVUK.analyticsGA4 || {}
```
So that it doesn't depend on being imported after another file has declared `window.GOVUK.analyticsGA4`?

This would never happen in our analytics implementation, since we have the page views etc. being imported first which would declare what `window.GOVUK.analyticsGA4` is. But if someone wanted to use this cookie consent pattern with our modules only, I think it would crash as `window.GOVUK.analyticsGA4` wouldn't exist.

Reply:
Yes, that makes sense, have added, thanks.